glrd/util.py

```diff
@@ -43,13 +77,18 @@ def extract_version_data(tag_name):
     return (int(match.group(1)), int(match.group(2))) if match else (None, None)
 
 def get_current_timestamp():
-    """Return the current timestamp."""
-    return int(datetime.now().timestamp())
+    """Get current timestamp in UTC."""
+    return int(datetime.now(pytz.UTC).timestamp())
```

Comment:
UNIX timestamps do not need to be UTC based as they are based on UTC anyway.

Reply:
:+1: